pupil-spa/src/app/app.module.ts

```diff
@@ -17,6 +17,7 @@ import {CheckCompleteComponent} from './check-complete/check-complete.component'
 import { HeaderComponent } from './header/header.component';
 import { FooterComponent } from './footer/footer.component';
 import { PhaseBannerComponent } from './phase-banner/phase-banner.component';
+import { StorageService } from './storage-service/storage.service'
```

Comment:
yes.  discussed this with Paula earlier.  Gut feeling is it will get very noisy if we don't have feature or category (component, service, module) folders.  I will flatten for now, but let's come back to this in a discussion asap.